src/server/index.js

```diff
@@ -43,7 +44,7 @@ const server = express();
 server
   .disable('x-powered-by')
   .use('/data', express.static(dataFolderToRender))
-  .use(express.static(publicDirectory))
+  .use(expressStaticGzip(publicDirectory))
```

Comment:
Since you're using `expressStaticGzip` - it would be useful to enable Brotli as well as gzip for the browsers that support them, with an order preference of brotli before gzip.

Assets compressed with Brotli have been shown to give 14-21% better compression compared with gzip - the range is depending on whether it's JS, CSS or HTML. 

Implementation details here: https://www.npmjs.com/package/express-static-gzip